validation/json-validator/src/main/java/com/redhat/mercury/util/schemavalidator/external/ExternalJsonSchemaValidator.java

```diff
@@ -13,37 +15,33 @@
  * A json schema validator that uses an external Apicurio registry that contains the schemas to validate against.
  * The actual validation is performed by everit json schema validator @see <a href="https://github.com/everit-org/json-schema">https://github.com/everit-org/json-schema</a>
  */
-public class ExternalJsonSchemaValidator {
+public class ExternalJsonSchemaValidator extends AbstructJsonSchemaValidator implements SchemaValidator {
 
     private RegistryClient client;
 
     /**
      * The constructor receives the Apicurio registry URL to work with.
      * The validator will use the regitry to fetch the appropriate schema to validate the json
      * @param apicurioRegistryUrl The Apicurio registry URL
-     */
-    public ExternalJsonSchemaValidator(String apicurioRegistryUrl) {
-        client = RegistryClientFactory.create(apicurioRegistryUrl);
-    }
-
-    /**
-     * When validating a schema the validator calls the Apicurio regitry to fetch the scheam.
-     * @param groupId The groupId of the schema in Apicurio, can be null, when null the default groupId is used.
+     * @param groupId The groupId of the schema in Apicurio, can be null.
      * @param artifactId The artifactId of the of the schema in Apicurio
-     * @param json The json to be validated against the schema
      */
-    public void validateSchema(String groupId, String artifactId, String json) {
-        if(groupId == null){
-            groupId = "default";
+    public ExternalJsonSchemaValidator(String apicurioRegistryUrl, String groupId, String artifactId) {
+        if(StringUtils.isBlank(apicurioRegistryUrl)){
```

Comment:
Don't use add another dependency that can potentially bring conflicts to other projects using this api just for the `StringUtils` do it with the `String` methods instead.
e.g.
```
if(apicurioRegistryUrl == null || apicurioRegistryUrl.trim().isEmpty()) {
```